### PR TITLE
Added foundation emails for issue #1523

### PR DESCRIPTION
**Pull request by sagarkaurav**
Added foundation emails scss to create html responsive email
screenshot from 2016-12-26 21-13-14
screenshot from 2016-12-26 21-14-23

### CODE REVIEW

**Comment by sonalkr132** [comment]
Hi @sagarkaurav. Thanks for the PR!

We would really prefer an inline CSS solution over adding gems. Checkout these resources:
* https://css-tricks.com/using-css-in-html-emails-the-real-story/
* http://stackoverflow.com/questions/4829254/best-practices-for-styling-html-emails

Also, please don't delete `database.yml.example`. Check out [Setting up the database](https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md#setting-up-the-database)

File: config/database.yml.example [hunk 1]
```diff
@@ -1,18 +0,0 @@
-development: &DEVELOPMENT
```
Comment (simi) [inline config/database.yml.example]: Why removed?

Comment (sagarkaurav) [inline config/database.yml.example]: I have added back :)

File: config/application.rb [hunk 2]
```diff
@@ -39,4 +40,5 @@ def self.config
 
   PROTOCOL = config['protocol']
   HOST = config['host']
+
```
Comment (simi) [inline config/application.rb]: :scissors:

File: test/helpers/email_helpers.rb [hunk 3]
```diff
@@ -5,4 +5,7 @@ def last_email_link
     link = /href="([^"]*)"/.match(body)
     link[1]
   end
+  def email_confirmation
```
Comment (simi) [inline test/helpers/email_helpers.rb]: Is this used?

Comment (sagarkaurav) [inline test/helpers/email_helpers.rb]: yes to it will fetch the frist user from database to create mail preview in development mode.

**Comment by sagarkaurav** [comment]
hi @sonalkr132 i updated the code and now its using inline css instead of gems
screen2

File: Gemfile.lock [hunk 4]
```diff
@@ -352,4 +352,4 @@ DEPENDENCIES
   yajl-ruby
 
 BUNDLED WITH
-   1.13.3
+   1.13.5
```
Comment (sonalkr132) [inline Gemfile.lock]: Please use bundler 1.13.3

**Comment by sonalkr132** [comment]
Please fix [rubocop errors](https://travis-ci.org/rubygems/rubygems.org/jobs/187516219#L1138) and [squash your commits](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642)

**Comment by sagarkaurav** [comment]
@sonalkr132 check out what else i need to fix. :)

**Comment by sonalkr132** [comment]
Hi @sagarkaurav Thanks for the effort :heart_decoration:, however you really need to clean up your commits. You don't need 12 commits for this nor you should be have merge commits in your PR. Check out following resources on rewriting git history:
* https://www.atlassian.com/git/tutorials/rewriting-history
* https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

If your are only getting started with git I would recommend reading chapter 1, 2, 3, 5 and 7 of [The pro git book](https://git-scm.com/book/en/v2). Feel free to give a shout out in #rubygem.org on [slack](https://bundler.slack.com) if you need help.

**Pull request closed without merging**